### DISCUSSION TITLE
Comparador

### DISCUSSION THREAD

**Discussion by GraceFinlay** [Telefonica/mistica-design#1020] [Proposals]
Buenas tardes,

Mi nombre es Grace y estoy trabajando en el proyecto de ecommerce global y os voy a indicar una serie de dudas relacionadas con Mística que han surgido en el desarrollo del proyecto.

# Contexto

El proyecto que está en marcha quiere incorporar unas mejoras en la tienda para que diferentes usuarios puedan comprar diferentes dispositivos. Estas mejoras que se quieren introducir son entre otras la mejora de los flujos de usuario y diferentes funcionalidades.

# Problema

Para este proyecto se requiere utilizar Mística y observamos que tenemos unas necesidades concretas en cuanto a [componentes.Es](http://componentes.es/) importante obtener feedback de vuestra parte con el fin de entender cómo podríamos implementar el proyecto usando Mística y qué cubra las necesidades planteadas.En este caso se quiere implementar la funcionalidad del comparador donde el usuario podrá comparar dos o mas dispositivos y evaluar cual le conviene más. 

# Propuesta

![Comparador empty](https://user-images.githubusercontent.com/99174134/214351902-2b3530e7-0d59-4df0-b2ee-ea2a57d953f2.png)
![Comparador 2 + 1](https://user-images.githubusercontent.com/99174134/214354575-7ef38950-2575-4dfa-9c9d-37df7144d04b.png)

Para el diseño de esta nueva funcionalidad se pensó:

1. Crear un componente tal y como podéis observar, en la primera imagen esta vacío y el usuario cuando va a buscar aparece un desplegable. Estos dropdown tan pequeños no logré visualizarlos en Mística. En la segunda imagen se muestra el componente ya tras la búsqueda.
2. Entendemos que estos componentes se podrían crear mediante átomos. 

Estamos abiertos a cualquier feedback que nos podáis dar para abordar el proyecto.

De antemano muchas gracias.

**Comment by yceballost** [comment]
Up
